src/runtime/account_loader.zig

```diff
@@ -78,32 +46,27 @@ fn Bank(comptime kind: BankKind) type {
                 .Mocked => self.inner.allocator,
             };
         }
```

Comment:
NIT: spaces between funcs 
```suggestion
        }
        
```

Reply:
fixed in ad41b539b9c9696ce4e727bf086dce6f741fd394